pkg/sql/alisa_submitter.go

```diff
@@ -112,20 +118,35 @@ func (s *alisaSubmitter) submit(program, alisaCode string) error {
 
 	// upload Alisa resource file to OSS
 	resourceName := randStringRunes(16)
-	bucket, err := getBucket(os.Getenv("SQLFLOW_ALISA_OSS_BUCKET"))
+	bucket, err := getBucket(os.Getenv("SQLFLOW_OSS_ENDPOINT"),
+		os.Getenv("SQLFLOW_OSS_AK"), os.Getenv("SQLFLOW_OSS_SK"), os.Getenv("SQLFLOW_OSS_BUCKET"))
 	if err != nil {
 		return err
 	}
 	if e := bucket.PutObjectFromFile(resourceName, filepath.Join(s.Cwd, tarball)); e != nil {
 		return err
 	}
-	defer func() {
-		bucket.DeleteObject(resourceName)
-	}()
+	defer bucket.DeleteObject(resourceName)
```

Comment:
maybe change `cleanUpModel` to `deleteOSSObject(path)` and call it here too?

Reply:
There are two OSS bucket configurations here: **Checkpoint Dir** which used in PAI-TF and **Alisa Resource Bucket** which used to store Alisa resource.

Maybe a clearer way is to initialize two `OSSClient` fields in `AlisaSubmitter` to operate OSS objects?

Reply:
Maybe use `deleteOSSObject(bucket, path)` should work?